src/main/java/org/jitsi/utils/dsi/DominantSpeakerIdentification.java

```diff
@@ -319,13 +318,51 @@ private static double computeSpeechActivityScore(
      */
     private final Map<T,Speaker> speakers = new HashMap<>();
 
+    /**
+     * The <tt>Speaker</tt>s in the multipoint conference with the highest
+     * current energy levels.
+     */
+    private final ArrayList<Speaker> loudest = new ArrayList<Speaker>();
+
     /**
      * Initializes a new <tt>DominantSpeakerIdentification</tt> instance.
      */
     public DominantSpeakerIdentification()
     {
     }
 
+    /**
+      * The number of current loudest speakers to keep track of.
+      */
+    private int numLoudestToTrack = 3;
```

Comment:
Can we default to `0` to avoid the calculations unless the users need them?

Reply:
the default jvb configuration was setting this to 0, but I agree that the module should have the correct default independent of who's using it. fixed.